Dockerfile

```diff
@@ -2,11 +2,13 @@ FROM python:3.9
 
 WORKDIR /app
 
-COPY requirements.txt requirements.txt
-RUN pip install -r requirements.txt
+COPY conftest.py requirements.txt /app/
```

Comment:
Why are we moving these to app folder?

Reply:
conftest.py is used to define some global configurations, such as pytest_plugins with the fixtures files (so they can be found by pytest automatically)
And requirements.txt is used to install the required packages to run the tests